web/src/app/monitors/monitors-list/monitors-list.component.html

```diff
@@ -0,0 +1,18 @@
+<mat-card>
+  <div fxLayout="row">
+    <div fxFlex="80%">
+      <h3>Monitors</h3>
+    </div>
+    <div fxFlex="20%"><button mat-raised-button type="submit" color="accent" routerLink="create">Add monitor</button></div>
+  </div>
+  <mat-list>
```

Comment:
Could these 2 sections be split into `mat-card-header` and `mat-card-content` similar to https://github.com/DashboardHub/PipelineDashboard/pull/1332/files#diff-c266124b5a3ac587f336bc3694c09d38

Reply:
sure